content/2019-07-16_hlavni-tema-sit.md

```diff
@@ -0,0 +1,73 @@
+Title: Hlavní téma - síť
+Date: 2019-07-16 15:32:00
+Modified: 2019-07-17 15:32:00
+Author: Karolina Surma
+
+
+## Tip na Markdown
+
+Sraz začal výjimečně připomínkou k formátování zpráv na blogu.
+Máme tendenci psát dlouhé odstavce, se kterými si git a Github neví dobře rady.
+Když něco v textu, který se táhne na několik řádků, změníme, Github už neoznačí konkrétní změnu, ale pro jistotu podsvítí celý ten řádek. Je proto lepší texty psát tak, ze každá věta bude na jednom řádku. Čtenář blogu si toho nevšimne, protože na stránkách se vše poskládá dobře, a my máme nástroj, jak kontrolovat i drobné změny textu.
+
+
+## Úprava interface: nechceme pořád posílat zprávy serveru
+
+Máme s interface problém: pořád posílá serveru zprávy. 
+I když hráč potvrdí svůj výběr, při každém zmáčknutí jakékoli klávesy, posílá serveru zprávu se svým stavem. 
+To se nám nelíbí a chceme to změnit. 
+Zde ale přichází na řadu aspekt, o němž jsme doposud vůbec neuvažovaly, tedy bezpečnost síťové komunikace.
+U komunikace server - klient chceme, aby server byl náš zdroj pravdy a byl ten prvek, který "šéfuje" všechno, co se k němu dostane. 
+Kdyby někdo napsal falešného klienta (a to nejen v Pygletu, ale třeba webového v JavaScriptu, nebo ovládání robot vytištěných na 3D tiskárně), který umí vyměnit karty za jiné nebo pozměnit nějaké atributy, nebo poslat přímo škodlivé zprávy, náš server musí vědět, že na takovou zprávu nemá reagovat. Pokud zpráva od klienta nebude dávat smysl, je lepší ji ošetřit na straně serveru. 
```

Comment:
> nebo ovládání robot vytištěných na 3D tiskárně

Nevím, co se tím myslí, takže to neumím upravit. Na 3D tiskárně má být vytištěné ovládání nebo roboti? 
Buď by tedy věta byla "nebo ovládání roboů vytištěné na 3D tiskárně" nebo "nebo ovládání robotů vytištěných na 3D tiskárně"

Reply:
Zkusila jsem to přepsat, snad teď dává tento kousek větší smysl :)